test/Interop/Cxx/reference/Inputs/module.modulemap

```diff
@@ -3,6 +3,11 @@ module Reference {
   requires cplusplus
 }
 
+module SpecialMembers {
```

Comment:
nit: why is this module case called `SpecialMembers`/`print-special-members.h`? The theme seems to be about logging copy- and move-constructor calls.

Reply:
In C++ we call move/copy constructors special member functions: https://en.wikipedia.org/wiki/Special_member_functions